### DISCUSSION TITLE
[FEATURE] 增加Docker 网络模式，便于安装应用时选择

### DISCUSSION THREAD

**Discussion by QYG2297248353** [1Panel-dev/1Panel#4234] [Ideas]
### 1Panel 版本

v1.10.1-lts

### 请描述您的需求或者改进建议

其一: 为了必要的功能
我想要同时安装 Emby与Jellyfin 但是他们端口都是一样的，就要做取舍。
例如：
如果要使用其提供的 投屏和网络发现 功能就要使用 Host主机模式部署。
使用主机模式且无法修改程序端口，就面临应用二选一，同类应用可以舍弃，但是如果是都必须安装的应用就无法同时安装。

其二: 断网应用
一些计算类应用，无需网络，可以忽略网络配置。或者一些不安全应用，不想使其获得网络。

其三：用户自定义创建的网络
用户在 容器 >> 网络 创建了网络，但是安装应用还要手动去修改应用的 `docker-compose.yml` 文件，操作麻烦。
虽然用户可以在启动后在容器列表编辑容器的方式修改网络，我还是希望能在安装时提供便利的选择。

期望功能：
希望在现有基础上增强docker容器网络。
在 vmware 虚拟机有一个桥接网络，可以实现虚拟机单独获取路由器IP与宿主机同网段IP。
查找了相关文献，虽然可以通过 创建网桥 然后使用 pipework 配置网络 指定网关与IP。但是其步骤较为麻烦，且需要 pipework 的擦插件支持。

### 请描述你建议的实现方案

- [ ] 安装应用时增加网络模式的选择
- [ ] 增强Docker容器的网络模式，实现 `桥接网络`
  - [ ] 桥接网络，允许指定静态IP

**Comment by wanghe-fit2cloud** [comment]
感谢反馈。
目前用户可以通过手动修改 compose 网络来满足这一需求。
对于 issue 中提到的自定义网络需求，我们可能无法在近期版本中实现，但会考虑在未来的 v2.0 版本中提供支持。望理解。
